discord/helpers.go

```diff
@@ -122,6 +120,23 @@ func getRoomAndRegionFromArgs(args []string) (string, string) {
 	return room, region
 }
 
+func getMemberFromString(s *discordgo.Session, GuildID string, input string) string {
+	// find which member the user was referencing in their message
+	// first check if is mentionned
+	ID, err := extractUserIDFromMention(input)
+	if err == nil {
+		return ID
+	}
+	members, _ := s.GuildMembers(GuildID, "", 500)
```

Comment:
Any reason on the 500 here?

Reply:
No specific reason. If I put 0 it would only return 1 member instead of being no limit as it should.